starlette_admin/statics/js/actions.js

```diff
@@ -0,0 +1,62 @@
+function submitAction(name, form, customResponse) {
+  let query = new URLSearchParams();
+  query.append("pks", id);
+  query.append("name", name);
+  let url = model.actionUrl + "?" + query.toString();
+  if (customResponse) {
+    if (form) {
+      form.action = url;
+      form.method = "POST";
+      form.submit();
+    } else {
+      window.location.replace(url);
+    }
+  } else {
+    $("#modal-loading").modal("show");
+    fetch(url, {
+      method: form ? "POST" : "GET",
+      body: form ? new FormData(form) : null,
+    })
+      .then(async (response) => {
+        await new Promise((r) => setTimeout(r, 500));
+        $("#modal-loading").modal("hide");
+        if (response.ok) {
+          successAlert((await response.json())["msg"]);
```

Comment:
`actions.js` is copied from `list.js`, with the table elements removed, and a few other changes. The detail page has no equivalent for `table.ajax.reload()`, but it would go here.

Some ideas:
- Refactor `submitAction` to include hooks where needed for detail / list actions
  - Ideal, but could get complicated
- Create an ajax `reload()` function for the detail page
- Refresh the page here, and deliver the success message after reload
- Respond with a redirect to the same page for actions coming from the detail page

Thoughts, @jowilf?

Reply:
@jowilf I implemented a solution using `localStorage`, where the success message is stored, and removed once the page reloads.

I even extended the pattern to the detail `delete` button, so it shows the success message after redirecting to the list page.

What do you think of this solution?

Reply:
It's perfect.